examples/Currency Conversion display.ipynb

```diff
@@ -0,0 +1,133 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Curency Conversions by [Sam2332](https://github.com/sam2332)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "import requests\n",
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starting_currency = 'USD'\n",
+    "ending_currency = 'CAD'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url = f\"http://www.bankofcanada.ca/valet/observations/FX{starting_currency}{ending_currency}/csv\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "ename": "ConnectionError",
+     "evalue": "HTTPConnectionPool(host='www.bankofcanada.ca', port=80): Max retries exceeded with url: /valet/observations/FXUSDCAD/csv (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fab9c1d49d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
+     "output_type": "error",
```

Comment:
Looks like you've committed this notebook with an error traceback - do you want to get rid of that?